package.json

```diff
@@ -1,74 +1,51 @@
-  "engines": {
-    "node": "12.x",
-    "npm": "6.14.4",
-    "yarn": "1.22.4"
-  },
-  "dependencies": {
-    "@fortawesome/fontawesome-svg-core": "1.2.28",
-    "@fortawesome/free-brands-svg-icons": "5.13.0",
-    "@fortawesome/free-solid-svg-icons": "5.13.0",
-    "@fortawesome/react-fontawesome": "0.1.10",
-    "@testing-library/jest-dom": "^4.2.4",
-    "@testing-library/react": "^9.3.2",
-    "@testing-library/user-event": "^7.1.2",
-    "@ungap/url-search-params": "0.2.2",
-    "axios": "0.20.0",
-    "expect": "26.5.0",
-    "fetch-mock": "9.10.7",
-    "node-fetch": "2.6.1",
-    "react": "^16.13.1",
-    "react-dom": "^16.13.1",
-    "react-redux": "7.2.0",
-    "react-router": "5.2.0",
-    "react-router-dom": "5.2.0",
-    "react-router-redux": "4.0.8",
-    "react-scripts": "3.4.1",
-    "redux": "4.0.5",
-    "redux-devtools-extension": "2.13.8",
-    "redux-thunk": "2.3.0"
-  },
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "jest --watchAll=false --coverage",
-    "coveralls": "jest --coverage && cat ./coverage/lcov.info | node node_modules/.bin/coveralls",
-    "eject": "react-scripts eject"
-  },
-  "jest": {
-    "transform": {
-      "^.+\\.[t|j]sx?$": "babel-jest"
-    },
-    "coveragePathIgnorePatterns": [
-      "/node_modules/",
-      "testconfig.js",
-      "package.json",
-      "package-lock.json",
-      "yarn.lock",
-      "serviceWorker.js"
-    ]
-  },
-  "eslintConfig": {
-    "extends": "react-app"
-  },
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
-  },
-  "devDependencies": {
-    "@babel/preset-env": "7.11.5",
-    "coveralls": "3.1.0",
-    "redux-mock-store": "1.5.4"
-  }
+	"name": "frontend",
+	"version": "0.1.0",
+	"private": true,
+	"dependencies": {
+		"@testing-library/jest-dom": "^5.11.5",
+		"@testing-library/react": "^11.1.2",
+		"@testing-library/user-event": "^12.2.2",
+		"react": "^17.0.1",
+		"react-dom": "^17.0.1",
+		"react-scripts": "4.0.0",
+		"web-vitals": "^0.2.4"
+	},
+	"scripts": {
+		"start": "react-scripts start",
+		"build": "react-scripts build",
+		"test": "react-scripts test",
+		"coveralls": "jest --coverage && cat ./coverage/lcov.info | node node_modules/.bin/coveralls",
+		"eject": "react-scripts eject"
+	},
+	"jest": {
+		"transform": {
+			"^.+\\.[t|j]sx?$": "babel-jest"
+		},
+		"coveragePathIgnorePatterns": [
+			"/node_modules/",
+			"testconfig.js",
+			"package.json",
+			"package-lock.json",
+			"serviceWorker.js"
+		]
+	},
+	"eslintConfig": {
+		"extends": [
+			"react-app",
+			"react-app/jest"
+		]
+	},
+	"browserslist": {
+		"production": [
+			">0.2%",
+			"not dead",
+			"not op_mini all"
+		],
+		"development": [
+			"last 1 chrome version",
+			"last 1 firefox version",
+			"last 1 safari version"
+		]
+	}
```

Comment:
Please use 2 line spacing and use space instead of tabs

Reply:
@devCayacoa the file would be named `.editorconfig` and would look like this:

```code
# EditorConfig is awesome: http://EditorConfig.org

# top-most EditorConfig file
root = true

# Unix-style newlines with a newline ending every file
[*]
charset = utf-8
indent_style = space
indent_size = 4
insert_final_newline = true
trim_trailing_whitespace = true


[**.html]
indent_style = space
indent_size = 4
wrap_line_length = 80
wrap_attributes = force

```

Reply:
The .editorconfig file takes care of the format for you. And yes, it will be applied to all the files